### PR TITLE
V2 release prep

### PR DESCRIPTION
**Pull request by jwildfire**
# Summary

Cleanup and dependency management for v2 release. 

# Tests

Quick sanity check to make sure app still runs as expected. You can also try loading and unloading `shinybusy` and `listviewer` to confirm that the functionality is added removed as expected.

### CODE REVIEW

File: R/app_startup.R [hunk 1]
```diff
@@ -8,7 +8,7 @@
...
 #' @param filterDomain domain used for the data/filter tab. Demographics ("`dm`") is used by default. Using a domain that is not one record per participant is not recommended. 
-#' @param chartSettingsPaths path(s) where customization functions are saved relative to your working directory. All charts can have itialization (e.g. myChart_Init.R) and static charts can have charting functions (e.g. myGraphic_Chart.R).   All R files in this folder are sourced and files with the correct naming convention are linked to the chart. See the Custom Charts vignette for more details. 
+#' @param chartSettingsPaths path(s) where customization functions are saved relative to your working directory. All charts can have ititialization (e.g. myChart_Init.R) and static charts can have charting functions (e.g. myGraphic_Chart.R).   All R files in this folder are sourced and files with the correct naming convention are linked to the chart. See the Custom Charts vignette for more details.
```
Comment (elimillera) [inline R/app_startup.R:11]:
```suggestion
#' @param chartSettingsPaths path(s) where customization functions are saved relative to your working directory. All charts can have initialization (e.g. myChart_Init.R) and static charts can have charting functions (e.g. myGraphic_Chart.R).   All R files in this folder are sourced and files with the correct naming convention are linked to the chart. See the Custom Charts vignette for more details. 
```

File: R/prepareChart.R [hunk 2]
```diff
@@ -1,4 +1,3 @@
-#' Prepare a chart object for safetyGraphcis
```
Comment (elimillera) [inline R/prepareChart.R:1]: Is the title removal intentional?

File: tests/testthat/test_mod_mappingColumn.R [hunk 3]
```diff
@@ -4,10 +4,12 @@ library(shinytest)
 library(testthat)
 library(stringr)
 
+skip_on_cran()
```
Comment (elimillera) [inline tests/testthat/test_mod_mappingColumn.R:7]: Are we having issues with tests on CRAN? I know this is becoming more popular and I generally agree with it but just curious.

Comment (jwildfire) [reply tests/testthat/test_mod_mappingColumn.R:7]: phantomjs isn't set up, so `shinytest` was bombing. There might be a workaround, but nothing obvious with a little googling.

**Pull request merged**